apache/zeppelin
# [ZEPPELIN-1486] Allow configuring whether shown values will be trimmed

### What is this PR for?
This patch improves Livy interpreter behavior: it allows configuring whether shown values will be trimmed or not.

### What type of PR is it?
Improvement

### Todos

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1486

### How should this be tested?
 * Set property `zeppelin.livy.spark.sql.truncateResult` to `true`
 * Run SQL query using Livy SQL interpreter
 * See whether long values are trimmed or not

### Screenshots (if appropriate)
![image](https://cloud.githubusercontent.com/assets/158011/18823498/580c162e-83c3-11e6-937e-b1c507aa4037.png)

### Questions:
* Does the licenses files need update? - No
* Is there breaking changes for older versions? - No
* Does this needs documentation? - Yes. Documenting `zeppelin.livy.spark.sql.truncateResult` option.
